pkg/cmd/step/create/test_data/step_create_task/add-env-to-default-in-buildpack/tasks.yml

```diff
@@ -20,6 +21,7 @@ items:
         name: version
       resources:
       - name: workspace
+        outputImageDir: ""
```

Comment:
where is that coming from? That's not related to the new label for meta and build pipeline, right?

Reply:
OOI, did you use the golden file mechanism to update the test files?